ES/ESmy020.xml

```diff
@@ -108,97 +431,229 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                           the punctuation signs; elements of Ethiopic numerals.&#xD; &#xD; The text
-                           on fol. 7r is written alternately in red and black lines.</seg>
+                        <seg type="rubrication">Divine names; incipits of the text sections; directive
+                          for the ministers; abbreviations, the words እሁነ, እግሌ or እግሊት, standing for
+                          the name of the deceased person or the one to be baptized; elements of Ethiopic
+                          numerals; elements of the punctuation signs. Rubrication has been executed in the main hand,
+                          along with the writing of the main text.</seg>
+                           <desc>Careful, slightly irregular; by a trained scribe.
+                             Broad, slightly right-sloping script.
+                           Parallel downstrokes, horizontal tops. Rectangualr letter shapes.
+                           The handwriting of <persName role="scribe" ref="PRS12076ZawaldaMaryam">Zawalda
+                              Māryām</persName> has some features which are well visible in <ref type="mss" corresp="ESmy020">Ms. MY-020</ref>
+                              and other manuscripts copied by him, such as <ref type="mss" corresp="ESmkl015">Ms. MKL-015</ref>,
+                              <ref type="mss" corresp="ESmkl016">Ms. MKL-016</ref>,
+                              <ref type="mss" corresp="ESqsm001">Ms. QSM-001</ref>
+                                <ref type="mss" corresp="ESsm014">Ms. SM-014</ref>, and a leaf
+                              Apart from the general appearance of the handwriting, some distinct features are: መ is very broad,
+                              with loops separated by a wide gap; ም has very short, nearly straight 6th order marker,
+                              set up in the middle of the loop; አ has the left and the right legs nearly parallel to each other;
+                              the numeral ፩ (1) is “closed” on the left side, not on the right side.
+                         </desc>
+
                      </handNote>
                   </handDesc>
                   <decoDesc>
 
-                     <decoNote type="frame" xml:id="d1">1) Chain of black and red dots (<locus target="#6ra">6ra</locus>)</decoNote>
-                     <decoNote type="frame" xml:id="d2">2) Chain of black and red dots (<locus target="#49ra">49ra</locus>)</decoNote>
-                     <decoNote type="frame" xml:id="d3">3) Three four dot asterisks, connected with
-                        lines; chain of black and red dots (<locus target="#53rb">53rb</locus>)</decoNote>
-                     <decoNote type="frame" xml:id="d4">4) Chain of black and red dots (<locus target="#71rb">71rb</locus>).</decoNote>
-                     <decoNote type="miniature" xml:id="d5"/>
+                    <decoDesc>
```

Comment:
you do not need a decoDesc around each decoNote, only one big decoDesc and then a list of decNotes
take all sub-level decoDesc tags out

Reply:
(this too makes the file invalid)